### PR TITLE
Add documentation for changes in the JS tracker 4.1

### PR DESCRIPTION
**Pull request by matus-tomlein**
Adds documentation for 2 new features in the release:

1. option to set the application version
2. new parameter in the context callback in button click tracking

### CODE REVIEW

File: docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/button-click/index.md [hunk 1]
```diff
@@ -212,6 +212,85 @@ enableButtonClickTracking({
...
+
+<Tabs groupId="platform" queryString>
+<TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+window.snowplow('enableButtonClickTracking', {
+  context: function (event, element) {
+    return {
+      schema: 'iglu:com.example_company/page/jsonschema/1-2-1',
+      data: { pageType: 'test' }
```
Comment (greg-el) [inline docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/button-click/index.md:269]: What do you think about including usage of one of the variables to make the dynamic usage a bit more obvious? maybe something like:

```js
return {
      schema: 'iglu:com.example_company/click/jsonschema/1-2-1',
      data: { content: element.textContent }
};
```

**Review by greg-el — APPROVED** [review]
Could of nits but otherwise LGTM!

File: docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracker-setup/initialization-options/index.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Here is a simple example of how to initialise a tracker, setting a few configura
 ```javascript
 snowplow('newTracker', 'sp', '{{collector_url_here}}', {
   appId: 'my-app-id',
+  appVersion: '0.1.0',
```
[the remark this answers is not in the document]
Comment (matus-tomlein) [reply docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracker-setup/initialization-options/index.md:29]: Yup, good catch!

File: docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/button-click/index.md (hunk 1 above)
Comment (matus-tomlein) [reply docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/button-click/index.md]: Good point! Have used that.

**Pull request merged**